### ISSUE TITLE
Installation failed Symfony 6.2.4 PHP 8.2.1

### ISSUE THREAD

**Issue by Papoel** [spatie/phpunit-watcher#154]
Hello,

The installation failed on my project, below is the screenshot of the error returned by my terminal.

```bash
composer require spatie/phpunit-watcher --dev
Cannot load Xdebug - it was already loaded
Deprecation Notice: Using ${var} in strings is deprecated, use {$var} instead in phar:///usr/local/bin/composer/src/Composer/Autoload/AutoloadGenerator.php:879
Deprecation Notice: Using ${var} in strings is deprecated, use {$var} instead in phar:///usr/local/bin/composer/src/Composer/Autoload/AutoloadGenerator.php:884
Info from https://repo.packagist.org: #StandWithUkraine
Using version ^1.23 for spatie/phpunit-watcher
./composer.json has been updated
Running composer update spatie/phpunit-watcher
Loading composer repositories with package information
Updating dependencies
Deprecation Notice: Using ${var} in strings is deprecated, use {$var} instead in phar:///usr/local/bin/composer/src/Composer/DependencyResolver/Problem.php:366
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - spatie/phpunit-watcher[1.23.5, ..., 1.23.6] require yosymfony/resource-watcher ^2.0 | ^3.0 -> satisfiable by yosymfony/resource-watcher[v2.0.0, v2.0.1, v3.0.0].
    - spatie/phpunit-watcher 1.23.0 requires php ^7.2 -> your php version (8.2.1) does not satisfy that requirement.
    - spatie/phpunit-watcher 1.23.1 requires symfony/console ^5.0 -> found symfony/console[v5.0.0, ..., v5.4.17] but it conflicts with your root composer.json require (6.2.*).
    - spatie/phpunit-watcher[1.23.2, ..., 1.23.4] require symfony/console ^5.2 -> found symfony/console[v5.2.0, ..., v5.4.17] but it conflicts with your root composer.json require (6.2.*).
    - yosymfony/resource-watcher v2.0.0 requires symfony/finder ^2.7|^3.0|^4.0 -> found symfony/finder[v2.7.0, ..., v2.8.52, v3.0.0, ..., v3.4.47, v4.0.0, ..., v4.4.44] but the package is fixed to v6.2.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - yosymfony/resource-watcher[v2.0.1, v3.0.0] require symfony/finder ^2.7|^3.0|^4.0|^5.0 -> found symfony/finder[v2.7.0, ..., v2.8.52, v3.0.0, ..., v3.4.47, v4.0.0, ..., v4.4.44, v5.0.0, ..., v5.4.17] but the package is fixed to v6.2.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires spatie/phpunit-watcher ^1.23 -> satisfiable by spatie/phpunit-watcher[1.23.0, ..., 1.23.6].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require spatie/phpunit-watcher:*" to figure out if any version is installable, or "composer require spatie/phpunit-watcher:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

To make sure that the problem did not come from my project I tried to install on three other projects :
  - Symfony 6.1 => Failed
  - Symfony 6.0 => Failed
  - Symfony 5.4 => Installation OK.

It seems that the problem is only present from symfony 6.0

Here are the details of my environment: 
 - Symfony project: 6.2.4
 - PHP version: 8.2.1
 - Symfony CLI version: 5.2.4

**Comment by vctrtvfrrr** [comment]
This tool seems to be abandoned. The maintainers don't seem to care about these errors.

**Comment by Papoel** [comment]
> This tool seems to be abandoned. The maintainers don't seem to care about these errors.

What a shame this bookstore looked so great...

**Comment by fabrizio-manca** [comment]
issue is still there
```
 Problem 1
    - Root composer.json requires yosymfony/resource-watcher ^3.0 -> satisfiable by yosymfony/resource-watcher[v3.0.0].
    - yosymfony/resource-watcher v3.0.0 requires symfony/finder ^2.7|^3.0|^4.0|^5.0 -> found symfony/finder[v2.7.0, ..., v2.8.52, v3.0.0, ..., v3.4.47, v4.0.0, ..., v4.4.44, v5.0.0, ..., v5.4.21] but the package is fixed to v6.3.0 (lock file version) by a part
ial update and that version does not match. Make sure you list it as an argument for the update command
```.

**Comment by Papoel** [comment]
I think it would be a good idea to revisit this project and make the necessary changes for it to work properly under Symfony 6.
